packages/paymasters/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@opengsn/paymasters",
+  "license": "MIT",
```

Comment:
GPL

Reply:
Right... And put a LICENSE file, I guess?..